docs/intro/overrides.rst

```diff
@@ -0,0 +1,387 @@
+    my_project.page_objects.DualExampleProductPage        my_project.page_objects.GenericProductPage  ['dualexample.com', 'dualexample.net']  []                                     500  {}
+
+Organizing Page Object Overrides
+--------------------------------
+
+After tackling the two (2) different approaches from the previous chapters on how
+to declare overrides, we can now explore how to organize them in our projects.
+Although it's mostly up to the developer which override declaration method to
+use. Yet, we'll present some approaches depending on the situation.
+
+To put this thought into action, let's suppose we are tasked to create a Page
+Object Project with overrides for eCommerce websites.
+
+Package-based Approach
+~~~~~~~~~~~~~~~~~~~~~~
+
+Using the **package-based** approach, we might organize them into something like:
+
+.. code-block::
+
+    my_page_obj_project
+    ├── cool_gadget_site
+    |   ├── us
+    |   |   ├── __init__.py
+    |   |   ├── products.py
+    |   |   └── product_listings.py
+    |   ├── fr
+    |   |   ├── __init__.py
+    |   |   ├── products.py
+    |   |   └── product_listings.py
+    |   └── __init__.py
+    └── furniture_shop
+        ├── __init__.py
+        ├── products.py
+        └── product_listings.py
+
+Assuming that we've declared the Page Objects in each of the modules to use the
+``default_registry`` like:
+
+.. code-block:: python
+
+    # my_page_obj_project/cool_gadget_site/us/products.py
+
+    from web_poet import handle_urls  # remember that this uses the default_registry
+    from web_poet.pages import ItemWebPage
+
+    @handle_urls("coolgadgetsite.com", overrides=GenericProductPage)
+    class CoolGadgetUsSiteProductPage(ItemWebPage):
+        def to_item(self):
+            ... # parsers here
+
+Then we could easily retrieve all Page Objects per subpackage or module like this:
+
+.. code-block:: python
+
+    from web_poet import default_registry
+
+    # We can do it per website.
+    rules = default_registry.get_overrides_from_module("my_page_obj_project.cool_gadget_site")
+    rules = default_registry.get_overrides_from_module("my_page_obj_project.furniture_site")
+
+    # It can also drill down to the country domains on a given site.
+    rules = default_registry.get_overrides_from_module("my_page_obj_project.cool_gadget_site.us")
+    rules = default_registry.get_overrides_from_module("my_page_obj_project.cool_gadget_site.fr")
+
+    # or even drill down further to the specific module.
+    rules = default_registry.get_overrides_from_module("my_page_obj_project.cool_gadget_site.us.products")
+    rules = default_registry.get_overrides_from_module("my_page_obj_project.cool_gadget_site.us.product_listings")
+
+    # Or simply all of Override rules ever declared.
+    rules = default_registry.get_overrides()
```

Comment:
I think for this to work user would need to call walk_modules

Reply:
Updated this on: https://github.com/scrapinghub/web-poet/pull/16/commits/daa3ff9bdfa754bacfd8a20862f68acb12a3fa61

Reply:
I've improved upon the previous commit linked above by creating a much more simpler interface for both `get_overrides()` + _**consuming/importing external packages/modules**_. Changes are in https://github.com/scrapinghub/web-poet/pull/16/commits/94fed0f1f8b1e2c8b62ec24d0b193b494ee8446f.